pkg/apply/apply.go

```diff
@@ -56,10 +56,12 @@ func NewApplierFromFile(path string, args *Args) (applydrivers.Interface, error)
 	}
 	currentCluster := cf.GetCluster()
 
+	runNewImages := GetImagesDiff(currentCluster.Spec.Image, cluster.Spec.Image)
```

Comment:
before you call GetImagesDiff() ，please use hashfunc to quick compare the spec!

Reply:
> before you call GetImagesDiff() ，please use hashfunc to quick compare the spec!

agree

Reply:
> > before you call GetImagesDiff() ，please use hashfunc to quick compare the spec!
> 
> agree

done